tests/integration/test_array_split.py

```diff
@@ -39,11 +40,42 @@
 ]
 
 
+class TestArraySplitErrors:
+    """
+    this class is to test negative cases
+    """
+
+    def test_indices_negative(self):
+        # negative indices should be not accepted
+        ary = np.arange(10)
+        # In numpy, it raises ValueError,
+        # msg is "number sections must be larger than 0."
+        # while in cunumeric, it passed.
+        num.array_split(ary, -2)
+
+    def test_indices_0(self):
+        # 0 indices should be not accepted
+        ary = np.arange(10)
+        # In numpy, it raises ValueError,
+        # msg is "number sections must be larger than 0.".
+        # In cunumeric, it raises ZeroDivisionError
```

Comment:
Is this difference intentional? If not, can the codepaths be updated to match numpy?

Reply:
I updated its code to match numpy.